lib/core/src/server/config.js

```diff
@@ -2,16 +2,12 @@ import path from 'path';
 import loadPresets from './presets';
 import serverRequire from './serverRequire';
 
-const noopWrapper = config => config;
-
 function customPreset({ configDir }) {
   return serverRequire(path.resolve(configDir, 'presets')) || [];
 }
 
 function getWebpackConfig(options, presets) {
-  const { configDir, wrapDefaultBabelConfig = noopWrapper } = options;
-
-  const babelOptions = presets.extendBabel({}, { configDir, wrapDefaultBabelConfig });
+  const babelOptions = presets.extendBabel({}, options);
```

Comment:
This breaks Babel 6 support, because it adds our defaults (which are Babel 7 only) to all configs

Reply:
We really need separate `extendDefaultBabel` and `extendBabel`

Reply:
Actually, react-docgen is probably the only usecase for the latter

Reply:
You mean `react-docgen` is the only working usecase?

Reply:
It's the only plugin that we need to add to any configuration because it typically isn't used in user app